File: PHP/uberX.php
<?php
require_once 'car.php';

class uberX extends car{
    public $brand;
    public $model;
    public $passenger;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function validarpasajeros($passanger)
    {
        if($passanger>=4){
            echo"El numero de pasajeros es correcto";
            $this->passenger=$passanger;
        }else{
            echo"El numero de pasajeros es incorrecto";
        }
    }
    public function printDataCar()
    {
        echo "
        Licencia: $this->license 
        Driver: {$this->driver->name} 
        Número de pasajeros: $this->passenger
        Marca: $this->brand
        Modelo: $this->model";
    }


}

?>

File: Java/src/UberX.java
public class UberX extends Car{
    String brand;
    String model;
    private Integer passenger;

    public UberX(String license, Account driver, String brand, String model) {
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    public void validarpasajeros(Integer passenger)
    {
        if(passenger>=4){
            System.out.println("El numero de pasajeros es correcto");
        }else{
            System.out.println("El numero de pasajeros es incorrecto");
        }
    }
    @Override
    void printDataCar() {
        System.out.println("License: " + license + " Driver: " + driver.document + " Nombre: " + driver.name + " Pasajeros:  " + passenger+" Marca: " + brand + " Modelo: " + model);
    }
}
